.gitignore

```diff
@@ -6,3 +6,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+.idea/
```

Comment:
instead of adding this to the project's `.gitignore` you should set up a global `.gitignore` local to your environment that ignores any IDE or workspace repos.  the project doesn't care and shouldn't know what environment you develop using 😉